src/field_theory/finite/basic.lean

```diff
@@ -0,0 +1,205 @@
+/-
+Copyright (c) 2019 Casper Putz. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Chris Hughes, Joey van Langen, Casper Putz
+-/
+
+import data.equiv.algebra
+import algebra.char_p
```

Comment:
Merging `field_theory.finite_card` with `field_theory.finite` breaks `data.zmod.quadratic_reciprocity` as in https://github.com/leanprover-community/mathlib/pull/819#issuecomment-472830821 since `algebra.char_p` imports `data.padics.padic_norm` which [has `attribute [class] nat.prime`](https://github.com/leanprover-community/mathlib/blob/81a31ca4a8c0287bf0b0ce40f1a0df16543b7abe/src/data/padics/padic_norm.lean#L48).

Reply:
Oops, I commented on the wrong commit, but the above holds true at 4f7e38a too.

Reply:
This is now fixed.